docker/experimental/Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN update-alternatives --install /usr/bin/clang++ clang++ $(which clang++-8) 70
 ENV CC=clang-8
 ENV CXX=clang++-8
 
-RUN pip install mkl mkl-include setuptools typing_extensions cmake requests
+RUN pip install setuptools typing_extensions cmake requests
```

Comment:
May have to create a conda environment to use the conda `nomkl` option to remove conda, something like https://stackoverflow.com/questions/46656367/how-to-create-an-environment-in-anaconda-with-numpy-nomkl